cnxarchive/sql/schema/functions.sql

```diff
@@ -141,3 +141,11 @@ AS $$
       plpy.warning(warning_messages)
   return content
 $$ LANGUAGE plpythonu;
+
+CREATE OR REPLACE FUNCTION StripHTML(html_text VARCHAR(255))
+  RETURNS text
+AS $$
+  from re import compile, sub
+  cleanr = compile('<.*?>') 
+  return sub(cleanr,'',html_text)
```

Comment:
Yes, I think the regular expression looks good!

Could you combine the two statements and do:

```
return re.sub('<.*?>', '', html_text)
```